modules/cluster/templates/config.hcl.tpl

```diff
@@ -32,7 +33,14 @@ listener "tcp" {
   tls_disable = 1
 }
 
-# Create an mTLS listener on the load balancer
+# Create non-TLS listener for the HTTP legacy health checks.  Make sure the VPC
+# firewall doesn't allow traffic to this port except from the probe IP range.
+listener "tcp" {
+  address     = "${lb_ip}:${vault_proxy_port}"
+  tls_disable = 1
+}
```

Comment:
This HTTP listener replaces nginx, the plain http health check remains for backwards compatibility but could be replaced with an https check.